README.md

```diff
@@ -0,0 +1,46 @@
+# External Secrets
+
+This operator reads information from a third party service
```

Comment:
I'd start with `The External Secrets Kubernetes operator`. 

What do you think?